### DISCUSSION TITLE
Why is can id limited to under 40?

### DISCUSSION THREAD

**Discussion by kkarnisky** [BroncBotz3481/YAGSL-Example#176] [Q&A] [answered]
I just recently started working with the yagswl using the latest version.  Just wondering why the latest version limits the can ids to under 40?  Thanks

**Comment by Technologyman00** [comment, marked as answer]
You are welcome to try higher numbers. There is nothing in the code that stops you. The reason that warning is provided is because when testing with Sparkmaxes I could make it go from working to not working just by changing the CAN ID. It may no longer be an issue but I haven't gotten around to do A-B testing again.

**Comment by kkarnisky** [comment]
Thank you very much for the quick response.  I was just concerned about the alerts. But good to know it doesn’t actually stop the code. KeithOn Feb 13, 2024, at 7:36 AM, Technologyman00 ***@***.***> wrote:﻿
You are welcome to try higher numbers. There is nothing in the code that stops you. The reason that warning is provided is because when testing with Sparkmaxes I could make it go from working to not working just by changing the CAN ID. It may not longer be an issue but I haven't gotten around to do A-B testing again.
